docs/pr_checklist.md

```diff
@@ -46,8 +46,10 @@ Closed PRs (for inspiration, previous sets of review comments will help you elim
 https://github.com/qmk/qmk_firmware/pulls?q=is%3Apr+is%3Aclosed+label%3Akeyboard
 
 - `info.json`
+    - with the move to [data driven](https://docs.qmk.fm/#/data_driven_config) keyboard configuration we encourage contributors to utilise as many features as possible of the info.json [schema](https://github.com/qmk/qmk_firmware/blob/master/data/schemas/keyboard.jsonschema)
     - valid URL
     - valid maintainer
+    - valid USB VID/PID and device version
     - displays correctly in Configurator (press Ctrl+Shift+I to preview local file, turn on fast input to verify ordering)
```

Comment:
Maybe something about the following being mandatory?
```suggestion
    - minimum implementation includes the following mandatory items
        - valid URL
        - valid maintainer
        - valid USB VID/PID and device version
        - displays correctly in Configurator (press Ctrl+Shift+I to preview local file, turn on fast input to verify ordering)
```

Reply:
Cool, have taken that on.